### ISSUE TITLE
crictl info fails with json: cannot unmarshal string into Go value

### ISSUE THREAD

**Issue by rgl** [kubernetes-sigs/cri-tools#1566]
#### What happened:

```console
# crictl info
FATA[0000] getting status of runtime: unmarshal status info JSON: json: cannot unmarshal string into Go value of type map[string]interface {} 
```

#### What you expected to happen:

It to work.

#### How to reproduce it (as minimally and precisely as possible):

1. Install containerd 1.7.20.
1. Execute `crictl info`.

#### Anything else we need to know?:

This worked fine with crictl 1.30.0.

#### Environment:

- Container runtime or hardware configuration: containerd 1.7.20.
- OS (e.g: `cat /etc/os-release`): ubuntu 22.04, but I think it does not matter.
- Kernel (e.g. `uname -a`): `Linux linux 5.15.0-113-generic #123-Ubuntu SMP Mon Jun 10 08:16:17 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux`
- Others:

**Comment by sij411** [comment]
I got the same issue when I installed containerd 1.7.19 and execute `sudo crictl info` on Ubuntu  22.04.

**Comment by saschagrunert** [comment]
@rgl @sij411 thank you for the report. Do you mind verifying that cri-tools b553776799fd278bc72d60dd8093e5633c7bf6f2 works for y'all?

**Comment by sij411** [comment]
> @rgl @sij411 thank you for the report. Do you mind verifying that cri-tools [b553776](https://github.com/kubernetes-sigs/cri-tools/commit/b553776799fd278bc72d60dd8093e5633c7bf6f2) works for y'all?
It didn't work for me :( 

Error message:
`FATA[0000] getting status of runtime: unmarshal status info JSON: json: cannot unmarshal string into Go value of type map[string]interface {} 
`

**Comment by sij411** [comment]
It didn't work for me. I tried commands on my oracle virtualbox. I'm not sure that hardware matters

Still got the same error message:
`FATA[0000] getting status of runtime: unmarshal status info JSON: json: cannot unmarshal string into Go value of type map[string]interface {} 
`

**Comment by saschagrunert** [comment]
Hm, I tested it with b5537767 and containerd  v1.7.20 and it outputs the JSON :thinking:

**Comment by sij411** [comment]
I used containerd v1.7.19. I think i should upgrade the version

**Comment by saschagrunert** [comment]
@sij411 it also works with containerd v1.7.19. Did you build this project manually and use the correct binary?

**Comment by saschagrunert** [comment]
Fixed with v1.31.1.

**Comment by rgl** [comment]
@saschagrunert its working. thank you!
